src/graphinfo.jl

```diff
@@ -63,7 +63,8 @@ function Model(;kwargs...)
     for (i, node) in enumerate(values(kwargs))
         @assert typeof(node) <: Tuple{Union{Array{Float64}, Float64}, Function, Symbol} "Check input order for node $(i) matches Tuple(value, function, kind)"
```

Comment:
At least you didn't compare by `==`, but this is even more preferable:

```suggestion
        @assert node isa Tuple{Union{Array{Float64}, Float64}, Function, Symbol} "Check input order for node $(i) matches Tuple(value, function, kind)"
```

Also: Is the restriction to arrays and floats hard-coded anywhere? I'd strongly prefer allowing _any_ type of value.